testData/preferences/maggie.json

```diff
@@ -0,0 +1,14 @@
+{
+    "flat": {
+        "contexts": {
+            "gpii-default": {
+                "name": "Default preferences",
+                "preferences": {
+                    "http://registry.gpii.net/common/magnification": 2.0,
+                    "http://registry.gpii.net/common/tracking": ["caret"],
```

Comment:
I think this is a windows issue since I can see that it's set to follow caret in the magnifier settings, but for whatever reason, my magnifier doesn't do so at all